### PR TITLE
Fix/4449 missing task content

### LINKED ISSUE #4449: v2.4 missing task content from v2.3 (mostly for cloud-only)
v2.4 missed tasks content updates added in v2.3.

##### Relevant URLs
- _Provide relevant URLs (the doc page in question, project issues, community threads, etc.)_

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #4449

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.4/process-data/get-started.md [hunk 1]
```diff
@@ -111,8 +139,56 @@ from(bucket: "example-bucket")
...
+```json
+{
+   "every": "1h",
+   "description": "Downsample host with 5 min precision.",
+   "name": "downsample_5m_precision",
+   "scriptID": "09b2136232083000"
+}
+```
+
+To create a script and a task that use parameters, see [Create a task to run an invokable script](/influxdb/cloud/process-data/manage-tasks/create-task/).
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/get-started.md:188]:
```suggestion
To create a script and a task that use parameters, see how to [create a task to run an invokable script](/influxdb/cloud/process-data/manage-tasks/create-task/).
```

File: content/influxdb/v2.4/process-data/get-started.md [hunk 2]
```diff
@@ -154,6 +230,43 @@ from(bucket: "example-bucket")
     |> to(bucket: "example-downsampled")
 ```
 
+{{% cloud-only %}}
+
+## Full example task with invokable script
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/get-started.md:235]: could we drop the term "Full" here?

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 3]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+
 - **Content-Type**: application/json
 - **Authorization**: Token *`INFLUX_API_TOKEN`*
 
 ##### Request body
+
 JSON object with the following fields:
 
-- **flux** : raw Flux task string
+- **flux** : raw Flux task string that contains a [`task` option](/flux/v0.x/spec/options/) and a query.
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:145]:
```suggestion
- **flux**: raw Flux task string that contains a [`task` option](/flux/v0.x/spec/options/) and a query.
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 4]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+    "cron": "0 * * * *",
+    "name": "downsample cpu",
+    "scriptID": "085a2960eaa20000",
+    "description": "This task downsamples CPU data every hour"
+}'
+```
+
+To create a task that passes parameters when invoking the script, pass the _`scriptParameters`_
+property in the request body.
+The following sample code creates a script with parameters and then creates a
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:209]:
```suggestion
The following sample code creates a script with parameters, and then creates a
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 5]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+
+echo $SCRIPT_ID
+
+curl https://cloud2.influxdata.com/api/v2/tasks \
+--header "Content-type: application/json" \
+--header "Authorization: Token INFLUX_API_TOKEN" \
+--data @- << EOF
+  {
+  "name": "30-day-avg-temp",
+  "description": "IoT Center 30d environment average.",
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:239]:
```suggestion
  "description": "IoT Center 30d temperature average.",
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 6]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+#### Request headers
+
+- **Content-Type**: application/json
+- **Authorization**: Token **`INFLUX_API_TOKEN`**
+
+#### Request body
+
+JSON object with the following fields:
+
+- **flux** : raw Flux task string that contains [`options`](/flux/v0.x/spec/options/) and the query.
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:272]:
```suggestion
- **flux**: raw Flux task string that contains [`options`](/flux/v0.x/spec/options/) and the query.
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 7]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+        "orgID": "INFLUX_ORG_ID",
+        "status": "active",
+        "description": "This task downsamples CPU data every hour"
+      }
+EOF
+```
+
+Replace the following:
+
+- **`INFLUX_API_TOKEN`**: your InfluxDB API token
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:299]:
```suggestion
- **`INFLUX_API_TOKEN`**: your InfluxDB [API token](/influxdb/cloud/security/tokens/view-tokens/)
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 8]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+- **`INFLUX_ORG`**: your InfluxDB organization name
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:300]:
```suggestion
- **`INFLUX_ORG`**: your InfluxDB [organization name](/influxdb/cloud/organizations/view-orgs/)
```

File: content/influxdb/v2.4/process-data/manage-tasks/create-task.md [hunk 9]
```diff
@@ -119,29 +127,177 @@ option task = {
...
+- **`INFLUX_ORG_ID`**: your InfluxDB organization ID
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/create-task.md:301]:
```suggestion
- **`INFLUX_ORG_ID`**: your InfluxDB [organization ID](/influxdb/cloud/organizations/view-orgs/#view-your-organization-id)
```

File: content/influxdb/v2.4/process-data/manage-tasks/delete-task.md [hunk 10]
```diff
@@ -24,12 +24,25 @@ related:
...
+
+_To find the task ID, see [how to view tasks](/influxdb/v2.4/process-data/manage-tasks/view-tasks/)_
+
+## Delete a task using the InfluxDB API
+
+Use the [`/tasks/TASK_ID` InfluxDB API endpoint](/influxdb/v2.4/api/#operation/DeleteTasksID) to delete a task and all associated records (task runs, logs, and labels).
+
+{{< api-endpoint method="DELETE" endpoint="http://localhost:8086/api/v2/tasks/TASK_ID" >}}
+
+_To find the task ID, see [how to view tasks](/influxdb/v2.4/process-data/manage-tasks/view-tasks/)_
```
Comment (kelseiv) [inline content/influxdb/v2.4/process-data/manage-tasks/delete-task.md:43]:
```suggestion
_To find the task ID, see [how to view tasks](/influxdb/v2.4/process-data/manage-tasks/view-tasks/)._
```

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.3/process-data/manage-tasks/task-run-history.md [hunk 11]
```diff
@@ -47,21 +47,25 @@ influx task run list --task-id=0000000000000000
...
+
+Use the [`/api/v2/tasks/TASK_ID/logs` InfluxDB API endpoint](/influxdb/v2.3/api/#operation/GetTasksIDLogs) to view the
+log events for a task and exclude additional task metadata.
+
+{{< api-endpoint method="GET" endpoint="http://localhost:8086/api/v2/tasks/TASK_ID/logs" >}}
+## View a task's run history with the InfluxDB API
+
+Use the [`/tasks/TASK_ID/runs` InfluxDB API endpoint](/influxdb/v2.3/api/#operation/GetTasksIDRuns) to view a task's run history.
+
+{{< api-endpoint method="GET" endpoint="http://localhost:8086/api/v2/tasks/{taskID}/runs" >}}
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/v2.3/process-data/manage-tasks/task-run-history.md:62]:
```suggestion
{{< api-endpoint method="GET" endpoint="http://localhost:8086/api/v2/tasks/TASK_ID/runs" >}}
```

File: content/influxdb/v2.3/process-data/manage-tasks/delete-task.md [hunk 12]
```diff
@@ -24,12 +24,25 @@ related:
...
 
 # Example
 influx task delete -i 0343698431c35000
 ```
+
+_To find the task ID, see [how to view tasks](/influxdb/v2.3/process-data/manage-tasks/view-tasks/)._
+
+## Delete a task using the InfluxDB API
+
+Use the [`/tasks/TASK_ID` InfluxDB API endpoint](/influxdb/v2.3/api/#operation/DeleteTasksID) to delete a task and all associated records (task runs, logs, and labels).
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/v2.3/process-data/manage-tasks/delete-task.md:39]: > shall we prefix with `/api/v2`across the board?
> 
> Recently saw a DAR where we updated some endpoints like this.

Probably, yes. I like the `api-endpoint` shortcode that Scott designed, but we need to devise a better pattern for using it with surrounding text. IMO, it ends up being repetitive b/c we need to explain it in text. Can we get away with not naming it in the paragraph?

**Pull request merged**